#include<iostream>
//#include<exeption>

using namespace std;

class CustomExeption: public exception

int main() {

}
